config/routes.rb

```diff
@@ -65,6 +65,9 @@
 
   resources :communities, only: [:new, :create]
 
+
+  devise_for :people, only: :omniauth_callbacks, controllers: { omniauth_callbacks: "sessions"}
```

Comment:
Can you quickly explain the effects and idea of pulling omniauth_callbacks here?